src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -314,6 +325,15 @@ def _configure_db_create_params(
         # Provisioning with capacity is not applicable to DataWarehouse
         arg_ctx.ignore('capacity')
 
+        # Provisioning with autoPauseDelay is not applicable to DataWarehouse
```

Comment:
Consider grouping these 3 under 1 comment that just says DW doesn't support serverless